package.json

```diff
@@ -442,6 +442,13 @@
         "title": "Submit Flink Statement",
         "category": "Confluent: Flink Statements"
       },
+      {
```

Comment:
Move up with other flink statement command buddies.

Reply:
Only concern here is that command ID change will affect telemetry